R/demUtils.R

```diff
@@ -0,0 +1,16 @@
+#' demUtils: A package with utility functions for common demographics data and estimation problems
+#'
+#' The demUtils package provides two categories of important functions:
+#' aggregation and scaling.
```

Comment:
plus additional utilities that assist with aggregating/scaling but have other more general applications as well?

Reply:
Think I made more general and we can add more sections if we add more groups of functions.